floodviz/static/js/map.js

```diff
@@ -95,22 +46,89 @@ document.addEventListener("DOMContentLoaded", function (event) {
 			.attr("transform", function (d) {
 				return "translate(" + projection(d.geometry.coordinates) + ")";
 			})
+			.attr("id", function(d) {
+				if (classname === 'gage-point') {
+					return 'map' + d.properties.id;
+				}
+			})
 			.attr("class", classname);
 		return (group);
-	}
-
+	};
 	/**
 	 * Add paths to the map
 	 * @param data The geojson to be added to the svg
 	 * @param classname The class to be given to each element for use in CSS
 	 */
-	function add_paths(data, classname) {
+	var add_paths = function (data, classname) {
 		var group = svg.append("g");
 		group.selectAll("path")
 			.data(data.features)
 			.enter()
 			.append("path")
 			.attr("d", path)
 			.attr("class", classname);
-	}
-});
\ No newline at end of file
+	};
+	
+	self.init = function() {
+		
+		var bounds = options.bounds;
```

Comment:
Not really necessary to declare variables for options properties, just use options.bounds, etc.